package.json

```diff
@@ -28,6 +28,7 @@
         "build:esm": "cross-env ENVIRONMENT=prod BUILD=esm rollup --config rollup.config.js",
         "build:stub": "cross-env ENVIRONMENT=prod BUILD=stub rollup --config rollup.config.js",
         "build:snippet": "uglifyjs snippet.js -m -o snippet.min.js",
+        "build:tests": "cross-env ENVIRONMENT=prod BUILD=tests rollup --config rollup.config.js",
```

Comment:
Not sure if you're missing an import or package somewhere. When I run this command after an `npm install` I get the following error:

```
TypeError: Cannot read properties of undefined (reading 'acorn')
```

Reply:
I have the wrong command here.  Will update.